data/json/monsters/mammal.json

```diff
@@ -1110,9 +1120,10 @@
     "fear_triggers": [ "SOUND", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
     "placate_triggers": [ "PLAYER_WEAK" ],
     "death_function": [ "NORMAL" ],
+    "harvest": "animal_medium",
```

Comment:
Lamb has size "SMALL", why give it harvest animal_medium? It needs to be a new category for animal_small